idea_town/frontend/static-src/.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
```

Comment:
Yeah, we might want to circle back & refine this too. I mainly added it out of annoyance because I indent 4-spaces in Python and 2-spaces in JS. Could add an .editorconfig in the root that handles both *.py and *.js